docs/version-specific/eb_list_toolchains.txt

```diff
@@ -14,7 +14,7 @@ cgmvapich2           Clang/GCC      MVAPICH2
 cgmvolf              Clang/GCC      MVAPICH2    OpenBLAS/LAPACK, ScaLAPACK(/BLACS), FFTW
 cgompi               Clang/GCC      OpenMPI
 cgoolf               Clang/GCC      OpenMPI     OpenBLAS/LAPACK, ScaLAPACK(/BLACS), FFTW
-dummy
+dummy                                           (Deprecated in favor of system toolchain)
```

Comment:
@justbennet All files in `docs/version-specific` are re-generated using `docs/scripts/update-all-docs.sh`, so this will be overwritten again.

You should change the script instead to inject this?

Reply:
OK.  I'll look at the script.  Sorry about that.